### DISCUSSION TITLE
Changing Window Title to display new font image

### DISCUSSION THREAD

**Discussion by Asreal-AJ** [NichtStudioCode/InvUI#77] [Q&A]
When using invUI, you set the menu title when creating the window. The challenge I'm facing is developing a card selection system that utilizes custom font images displayed by a texture pack. I need the ability to dynamically change the inventory name to showcase the new font images whenever the player clicks the "next" button.

I've been trying different ways to make it work but I cannot find a solution, any suggestions?

**Comment by NichtStudioCode** [comment]
Are you looking for Window#changeTitle?

**Reply by Asreal-AJ** [reply]
Yes but for the item, I'm looking for a way the item itself can change the window title to a different character(eg.`\uA001->\uA002`) when interacted with. So when a player clicks a next item in the inventory, the item gets the next character to display and changes the inventory title accordingly.

**Reply by Asreal-AJ** [reply]
Any idea how to do that? ^

**Reply by NichtStudioCode** [reply]
You call Window#changeTitle with the new title when the player clicks the item. Not sure what more to tell you here.
